txdbus/router.py

```diff
@@ -45,10 +45,8 @@ def match(self, m):
                     return
 
             if hasattr(self, 'path_namespace'):
-                # See https://github.com/cocagne/txdbus/issues/62
-                # FIXME: Skipping validation for this since we already have an
-                # open issue for it:
-                if m.path is None or not m.path.startswith(pn):  # noqa
+                if (m.path is None or
+                        not m.path.startswith(self.path_namespace)):
```

Comment:
I thought about that, when flake8 complained wrt my first commit. Then I got somehow lost in reading the details about E125 vs E129 for example in https://github.com/PyCQA/pycodestyle/issues/126. I ended up going for one possible, valid approach. But the one you suggest sounds nice, so here it is.